packages/sdk/src/contracts/bridge.contract.ts

```diff
@@ -122,7 +120,7 @@ export class Bridge {
    * @returns The HolographFactory contract address in the provided network.
    */
   async getFactory(chainId: number) {
-    return this._getContractFunction(chainId, 'getFactory')
+    return this._getContractFunction({chainId, functionName: 'getFactory'})
```

Comment:
should have return type

Reply:
yep agree, but I think we should address this in another ticket. Because this is happening with all functions.